data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,405 @@
+				action
+					set "incipias: helped the derelict"
+		personality
+			confusion 50
+			derelict staying uninterested
+	on enter "Hui'uc"
+		conversation
+			`On entering the system you find an alien spaceship disabled and badly damaged. There do not appear to be any other ships or inhabited planets in the system.`
+
+mission "Incipias: Help The Stranded 1"
+	landing
+	source
+		government "Quarg (Incipias)"
+	destination "Pon'tes"
+	to offer
+		has "incipias: helped the derelict"
+	on offer
+		conversation
+			`Maybe the Quarg could help you with the stranded ship.`
+			choice
+				`	(Tell them about it.)`
+				`	(Not right now.)`
+					defer
+			`	Right after you arrive on the planet, you search for a Quarg who speaks your language so that you can tell them about the stranded ship you found. One of the Quarg seems to understand your intention and gestures for you to wait. It returns a few hours later accompanied by the same Quarg you talked to before.`
+			`	"Human, you seem to be in quite a haste. What could be such a pressing matter?" it asks.`
+			choice
+				`	(Explain the situation.)`
+					goto explanation
+				`	"I need advice regarding the species in this region."`
+			`	It looks at you curiously. "So you have held council with the young Incipias? This one is nonplussed that you ably formed correspondence. It took a prolonged period of time even for us to decipher their way of dissemination."`
+			`	You quickly explain what happened. From what you can tell, the Quarg seems relieved that you were not able to talk to them.`
+			label explanation
+			`	"You ran into a most peculiar situation. Surpassing the borders of their system is not yet a capability possessed by the Incipias." It tilts its head for a moment, considering what you just said. "It is our desire to help them, but it is yet premature for us to reveal ourselves to their species, and certainly premature for them to be interacting with independent interlopers such as yourself. I fear that we are impotent in this unfortunate turn of events."`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"I understand. I'll leave them alone."`
+					goto declined
+				`	"How can you make such a cold-hearted decision?"`
+			`	"It is perhaps a laborious task for an adolescent species such as yourself to comprehend our ways, but we must place the continuation of species above the existence of individuals. If we reveal ourselves to them at this inopportune moment, the shock of such an advanced species appearing in their space will fan the flames of chaos."`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"You can deal with the consequences later. Help them now!"`
+			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."`
```

Comment:
```suggestion
			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, this one can foresee the likelihood of you making a rash and impetuous decision to contact these Incipias further."`
```
Consistency of "this one" vs "I". Also, you already contacted the ship, so contacting it further beyond that would be the concern.